packages/strapi-design-system/vite.config.ts

```diff
@@ -15,7 +16,7 @@ export default defineConfig(async () => {
       exclude: [],
     },
     build: {
-      emptyOutDir: false,
+      emptyOutDir: process.env.DTS === 'true',
```

Comment:
I think this will break storybook when you are developing i.e. you'll make a change in the DS and it'll empty the folder and storybook will crash.

Reply:
ok